src/subaddress.php

```diff
@@ -70,9 +76,9 @@ private function ge_scalarmult_base($scalar)
 	 */
 	private function keccak_256($message)
 	{
-		$keccak256 = SHA3::init(SHA3::KECCAK_256);
-		$keccak256->absorb(hex2bin($message));
-		return bin2hex($keccak256->squeeze(32)) ;
+		$message_bin = hex2bin($message);
+        $hash = keccak::hash($message_bin, 256);
```

Comment:
I'm seeing something weird with indentation, is it correct? It does not seem so.

Reply:
It confused me as well. It was a mix of tabs and spaces.

Reply:
Thanks for solving it!